doc/manual/rl-next/self-lfs-attr.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Can you combine this with the release note for git-lfs (./doc/manual/rl-next/git-lfs-support.md)?